Meta/download_file.py

```diff
@@ -72,10 +73,14 @@ def main():
 
     os.makedirs(output_file.parent, exist_ok=True)
 
-    with urllib.request.urlopen(args.url) as f:
+    request = urllib.request.Request(args.url, headers={"Accept-Encoding": "gzip"})
+    with urllib.request.urlopen(request) as f:
         try:
             with tempfile.NamedTemporaryFile(delete=False, dir=output_file.parent) as out:
-                out.write(f.read())
+                if "Content-Encoding" in f.headers and f.headers["Content-Encoding"] == "gzip":
```

Comment:
nit: Does `f.headers.get('Content-Encoding') == 'gzip'` work? I think `get()` will return None if the key doesn't exist, and then the comparison is false.